pydicom/pixel_data_handlers/rle_handler.py

```diff
@@ -0,0 +1,219 @@
+import sys
+import pydicom.uid
+import pydicom.encaps
+from struct import unpack
+have_numpy = True
+try:
+    import numpy
+except ImportError:
+    have_numpy = False
+    raise
+
+sys_is_little_endian = (sys.byteorder == 'little')
+
+RLESupportedTransferSyntaxes = [
+    pydicom.uid.RLELossless,
+]
+
```

Comment:
You've defined this twice, just import `RLECompressedLosslessSyntaxes` from `pydicom.uid`